### PR TITLE
i3[-gaps]: new package at 4.19

### PR DESCRIPTION
**Pull request by Ominitay**
## Description of package
i3 is a popular tiling window manager. Both of these packages are virtually identical (only difference is where they download their source from). 

## Installed manifest of package

`kiss manifest i3[-gaps]`

```
/var/db/kiss/installed/i3[-gaps]/version
/var/db/kiss/installed/i3[-gaps]/sources
/var/db/kiss/installed/i3[-gaps]/patches/i3-no-libstartup-notification.patch
/var/db/kiss/installed/i3[-gaps]/patches/
/var/db/kiss/installed/i3[-gaps]/manifest
/var/db/kiss/installed/i3[-gaps]/depends
/var/db/kiss/installed/i3[-gaps]/checksums
/var/db/kiss/installed/i3[-gaps]/build
/var/db/kiss/installed/i3[-gaps]/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/local/share/xsessions/i3.desktop
/usr/local/share/xsessions/i3-with-shmlog.desktop
/usr/local/share/xsessions/
/usr/local/share/doc/i3/wsbar.png
/usr/local/share/doc/i3/two_terminals.png
/usr/local/share/doc/i3/two_columns.png
/usr/local/share/doc/i3/tree-shot4.png
/usr/local/share/doc/i3/tree-shot3.png
/usr/local/share/doc/i3/tree-shot2.png
/usr/local/share/doc/i3/tree-shot1.png
/usr/local/share/doc/i3/tree-layout2.png
/usr/local/share/doc/i3/tree-layout1.png
/usr/local/share/doc/i3/snapping.png
/usr/local/share/doc/i3/single_terminal.png
/usr/local/share/doc/i3/refcard_style.css
/usr/local/share/doc/i3/refcard.html
/usr/local/share/doc/i3/modes.png
/usr/local/share/doc/i3/logo-30.png
/usr/local/share/doc/i3/layout-saving-1.png
/usr/local/share/doc/i3/keyboard-layer2.png
/usr/local/share/doc/i3/keyboard-layer1.png
/usr/local/share/doc/i3/i3-sync.png
/usr/local/share/doc/i3/i3-sync-working.png
/usr/local/share/doc/i3/bigpicture.png
/usr/local/share/doc/i3/
/usr/local/share/doc/
/usr/local/share/applications/i3.desktop
/usr/local/share/applications/
/usr/local/share/
/usr/local/include/i3/ipc.h
/usr/local/include/i3/
/usr/local/include/
/usr/local/etc/i3/config.keycodes
/usr/local/etc/i3/config
/usr/local/etc/i3/
/usr/local/etc/
/usr/local/bin/i3bar
/usr/local/bin/i3-with-shmlog
/usr/local/bin/i3-sensible-terminal
/usr/local/bin/i3-sensible-pager
/usr/local/bin/i3-sensible-editor
/usr/local/bin/i3-save-tree
/usr/local/bin/i3-nagbar
/usr/local/bin/i3-msg
/usr/local/bin/i3-migrate-config-to-v4
/usr/local/bin/i3-input
/usr/local/bin/i3-dump-log
/usr/local/bin/i3-dmenu-desktop
/usr/local/bin/i3-config-wizard
/usr/local/bin/i3
/usr/local/bin/
/usr/local/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

**Comment by Ominitay** [comment]
This is just copied over from the main repo, and is now out of date (latest version is 4.19.1). I currently don't have an instance of Kiss running, and so will update it once I have that set up.

File: community/i3-gaps/build [hunk 1]
```diff
@@ -0,0 +1 @@
+../i3/build
```
Comment (mmatongo) [inline community/i3-gaps/build:1]: This is not a valid `buildfile`.  It should contain the proper build instructions of the application.

File: community/i3-gaps/depends [hunk 2]
```diff
@@ -0,0 +1 @@
+../i3/depends
```
Comment (mmatongo) [inline community/i3-gaps/depends:1]: This file is also not valid, it should contain the proper dependencies of the application.

File: community/i3-gaps/patches [hunk 3]
```diff
@@ -0,0 +1 @@
+../i3/patches
```
Comment (mmatongo) [inline community/i3-gaps/patches:1]: This should be a folder of the actual patches dir.

**Review by mmatongo — CHANGES_REQUESTED** [review]

File: community/i3-gaps/patches (hunk 3 above)
Comment (Ominitay) [reply community/i3-gaps/patches:1]: These are all symlinks. The previous package utilised symlinks, so I was under the impression that they were supported or allowed. I'll replace them with duplicated files from `i3` if you wish.

**Comment by mmatongo** [comment]
Did you use `kiss-link`?

**Comment by Ominitay** [comment]
I though that `kiss-link` is for linking between repositories, not packages?

**Comment by dilyn-corner** [comment]
`kiss-link` will search `KISS_PATH` for `$pkg/$file` and symlink `$file` to the first match it finds. `kiss-link` was implemented so that if you fork a package to change `build`, for instance, you don't also need to track updates; the upstream repository you `kiss-fork $pkg` from will do that for you. 
If $/kisslinux/community had no issues with such a symlink, we shouldn't either. That being said, `i3-gaps` is already in this repository?

**Comment by Ominitay** [comment]
> That being said, i3-gaps is already in this repository?
That's strange... it hasn't been merged into the other, and this is the only PR that was done for it here as well. Did @git-bruh merge my PR before I made this duplicate?

**Comment by git-bruh** [comment]
Yeah i merged it and fixed up symlinks

**Comment by mmatongo** [comment]
I always used it for packages within and across.

If it's what the official repos went with then don't mind me.

**Comment by Ominitay** [comment]
> Yeah i merged it and fixed up symlinks

Cool! I'll make a new PR to update it when I have the time.

**Pull request closed without merging**